### PR TITLE
Polls now support voting more than one option

### PR DESCRIPTION
**Pull request by juho-jaakkola**
- [x] Add German translations
 - [x] Allow number of votes to be same as number of options

### CODE REVIEW

File: classes/Poll.php [hunk 1]
```diff
@@ -247,4 +258,16 @@ public function getResponseCount() {
...
+	 * Get amount of people who have voted
+	 *
+	 * @return int
+	 */
+	public function getVoterCount() {
+		// Make sure the values have been populated
+		$this->fetchResponses();
+
+		return $this->voter_count;
+	}
```
Comment (juho-jaakkola) [inline classes/Poll.php]: This new method allows you to get the number of voters instead of number of individual votes.

File: languages/de.php [hunk 2]
```diff
@@ -61,6 +61,10 @@
 	'poll:front_page_label' => "Diese Umfrage zur aktuellen \"Umfrage der Stunde\" machen",
 	'poll:open_poll_label' => "Zeige in den Ergebnissen, welche Mitglieder für was gestimmt haben (offene Umfrage)",
 	'poll:show_voters' => "Zeige Umfrageteilnehmer",
+	'poll:max_votes:label' => "Anzahl maximale Stimmen je Teilnehmer",
+	'poll:max_votes:desc' => "Allowed number of votes per participant",
+	'poll:max_votes:exceeded' => "Number of allowed votes must be less than the number of options",
```
Comment (juho-jaakkola) [inline languages/de.php]: Let me know the translations for these two, and I'll add them to the PR.

**Comment by iionly** [comment]
Shouldn't the maximum number of allowed votes be less or equal the number of poll options? Sometimes it might make sense to check all offered poll options.

German translations for new strings:

'poll:max_votes:label' => "Maximal mögliche Anzahl an Stimmen pro Teilnehmer der Umfrage",
'poll:max_votes:desc' => "Dies ist Maximalzahl an Auswahlmöglichkeiten, die ein Teilnehmer der Umfrage bei der Stimmabgabe selektieren kann.",
'poll:max_votes:exceeded' => "Die maximale Anzahl an Stimmen muss kleiner als die Anzahl der Auswahlmöglichkeiten der Umfrage sein.",
'poll:max_votes:info' => "Du hast maximal %s Stimmen.",

In case the maximum number of votes allowed is equal the number of poll options the translations of one string would be slightly different:

'poll:max_votes:exceeded' => "Die maximale Anzahl an Stimmen kann nicht größer als die Anzahl an Auswahlmöglichkeiten der Umfrage sein.",


Please give me some time for merging. I don't think I have time at all before the weekend.

**Comment by juho-jaakkola** [comment]
Missing translations have been added. Also it is now possible to have as many votes as there are options.

**Comment by iionly** [comment]
@juho-jaakkola 
I've reworked your PR and finally added the support for multi-answer polls with https://github.com/iionly/poll/commit/6ba6385ba00a447c8de6c6bc1c294ce6b73fe58d.

I've added an additional plugin setting to allow for enabling / disabling the multi-answer support. The changes (also in the other recent commits) made some modifications in the language files necessary. If you would like to update the Finnish translations I will surely wait a few days before making a release. I also want to rework the code of the plugin settings view (including separating the hints / instructions from the label strings and display them as subtext instead). I'll try to do that tomorrow (i.e. language files will again change).

**Pull request closed without merging**